### PR TITLE
Wizard recipe: fakechroot-v2.20.2

### PR DESCRIPTION
**Pull request by Keno**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: fakechroot
* Version: v2.20.2

@staticfloat please review and merge.

### CODE REVIEW

**Comment by Keno** [comment]
musl is disabled because it gets mis-detected by configure. Also the version I used is 2.20.2, because it's master form 2.20.1, but there is no version 2.20.2

File: F/fakechroot/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dex4er/fakechroot.git", "b42d1fb9538f680af2f31e864c555414ccba842a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/fakechroot
+./autogen.sh 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```
Comment (staticfloat) [inline F/fakechroot/build_tarballs.jl:17]: Try `--enable-shared` or something to get the shared library?

Comment (giordano) [reply F/fakechroot/build_tarballs.jl:17]: Isn't it `lib/fakechroot`?

Comment (staticfloat) [reply F/fakechroot/build_tarballs.jl:17]: It looks to me like it's only installing the static library, e.g. `libfakechroot.a`

Comment (staticfloat) [reply F/fakechroot/build_tarballs.jl:17]: Nope, I'm totally wrong.  Yeah, we just need to update the search path for the `LibraryProduct`.

File: F/fakechroot/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libfakechroot", :libfakechroot)
```
Comment (staticfloat) [inline F/fakechroot/build_tarballs.jl:35]:
```suggestion
    LibraryProduct("libfakechroot", :libfakechroot, ["lib/fakeroot"])
```

File: F/fakechroot/build_tarballs.jl (hunk 1 above)
Comment (giordano) [reply F/fakechroot/build_tarballs.jl:17]: I assume the wizard should catch this though.

Comment (giordano) [reply F/fakechroot/build_tarballs.jl:17]: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/745

File: F/fakechroot/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libfakechroot", :libfakechroot, ["lib/fakeroot"])
```
Comment (giordano) [inline F/fakechroot/build_tarballs.jl:35]:
```suggestion
    LibraryProduct("libfakechroot", :libfakechroot, ["lib/fakechroot"])
```
:stuck_out_tongue:

**Pull request merged**